peewee_sanic_rest/filters.py

```diff
@@ -126,6 +126,10 @@ class StringFilter(TypeFilter):
     type = str
 
 
+class BooleanFilter(TypeFilter):
+    type = bool
```

Comment:
You should explicitly check for truthy values, like `1`, `true` etc. Not sure about final list, but right now `bool('0') == true`. Remember that we receive strings from a request.

Another solution is to use `IntegerFilter` and treat all values except `0` as truthy.